src/elm/Page/Transfer.elm

```diff
@@ -0,0 +1,414 @@
+    let
+        t =
+            I18Next.t shared.translations
+
+        head =
+            case state of
+                Received ->
+                    div [ class "flex flex-row" ]
+                        [ div [ class "pr-0 pl-1 py-2 m-2 " ]
+                            [ div [ class "border border-solid border-green border-t-0 border-r-3 border-b-3 border-l-0 inline-block p-1 sm:ml-5 ml-10 mt-5 rotate-135" ]
+                                []
+                            ]
+                        , div [ class "pl-3 pr-3 py-2 m-2" ]
+                            [ hr [ class "sm:-ml-6 -ml-5 items-center border border-dashed border-green m-auto w-6 mt-6" ] [] ]
+                        ]
+
+                _ ->
+                    div [ class "flex flex-row" ]
+                        [ div [ class "px-4 py-2 m-2" ]
+                            [ hr [ class "ml-5 border border-dashed border-green w-8 mt-6 m-auto mb-6" ] [] ]
+                        ]
+
+        tail =
+            case state of
+                Received ->
+                    div [ class "flex flex-row" ]
+                        [ div [ class "pl-2 pr-10 py-2 m-2" ]
+                            [ hr [ class "sm:-ml-8 -ml-10 border border-dashed border-green sm:w-8 w-6 mt-6 m-auto mb-6" ] [] ]
+                        ]
+
+                _ ->
+                    div [ class "flex flex-row" ]
+                        [ div [ class "pl-2 pr-2 py-2 m-2" ]
+                            [ hr [ class "-ml-8 items-center border border-dashed border-green w-6 mt-6 m-auto mb-6" ] [] ]
+                        , div [ class "px-4 py-2 m-2" ]
+                            [ div [ class "border border-solid border-green border-t-0 border-r-3 border-b-3 border-l-0 inline-block p-1 -rotate-45 -ml-12 mt-5" ]
+                                []
+                            ]
+                        ]
+    in
+    div [ class "-ml-16 flex flex-row mt-5" ]
+        [ head
+        , div [ class "px-4 py-2 m-2" ]
+            [ div [ class "-ml-10 border border-solid rounded border-green bg-white" ]
+                [ div [ class "ml-1" ]
+                    [ p [ class "pt-1 text-caption font-hairline text-gray-900 pl-1" ]
+                        [ text <|
+                            case state of
+                                Received ->
+                                    String.toUpper (t "transfer_result.received")
+
+                                Transferred ->
+                                    String.toUpper (t "transfer_result.transferred")
+
+                                NotInvolved ->
+                                    String.toUpper (t "transfer_result.transferred")
+                        ]
+                    , div [ class "flex flex-row" ]
+                        [ p [ class "pl-1 pr-5 font-medium text-green" ]
+                            [ text <|
+                                String.fromFloat transfer.value
```

Comment:
the font size should be bigger. check the print from figma and the implementation:

<img width="535" alt="Screen Shot 2019-12-19 at 13 37 32" src="https://user-images.githubusercontent.com/1082127/71191223-bf503480-2264-11ea-8cfa-6c9ca990bd14.png">
![Screen Shot 2019-12-19 at 13 37 37](https://user-images.githubusercontent.com/1082127/71191224-bf503480-2264-11ea-8ea2-6f35738ea249.png)